model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -74,6 +74,7 @@ data class AnalyzerResult(
             it.dependencies.forEach { addErrors(it) }
         }
 
-        return collectedErrors.filterValues { it.isNotEmpty() }.mapValues { it.value.distinct() }
+        return collectedErrors.filterValues { it.isNotEmpty() }.mapKeys { "${project.id.provider}:${it.key}" }
+                .mapValues { it.value.distinct() }
```

Comment:
This starts to get a bit long, and mapping keys / values in two separate runs seems a bit inefficient. Maybe this is better (but I'm not sure myself):

        val mappedErrors = mutableMapOf<String, List<String>>()
        collectedErrors.forEach { key, value ->
            if (value.isNotEmpty()) {
                mappedErrors["${project.id.provider}:$key"] = value.distinct()
            }
        }
        return mappedErrors

Reply:
Done.